### ISSUE TITLE
Cannot run Spike example

### ISSUE THREAD

**Issue by vankhoa21991** [CEA-LIST/N2D2#30]
Hello,
I tried to run the example to convert CNN to SNN (LeNet), by these two commands:
../build/bin/n2d2 "../models/LeNet.ini" -learn 6000000 -log 100000
../build/bin/n2d2 "../models/LeNet_Spike.ini" -test

And I ran into this error:
Option -test: perform testing
Loading network configuration file ../models/LeNet_Spike.ini
Layer: conv1 [Conv(Transcode_CUDA)]
Notice: Could not open configuration file: conv1.cfg
Notice: Unused section fc2.Target in INI file
Notice: Unused section fc2.config in INI file
Time elapsed: 0.882318 s
Error: Cell_Spike::addInput(): mapping length must be equal to the number of outputs

Do you know where does it come from?
Thanks

**Comment by vankhoa21991** [comment]
The same happens with mnist24_16c4s2_24c5s2_150_10.ini

**Comment by olivierbichler-cea** [comment]
Hello,
Thanks for your feedbacks, this was a bug that should be corrected in the latest commit 20be5f401c15802a4de08bcd649f4b84e10ce223
Please tell us if everything is all right now.

**Comment by vankhoa21991** [comment]
Thanks for your response, i get the latest commit and it show new errors:

../build/bin/n2d2 "../models/LeNet.ini" -learn 6000000 -log 100000
Option -log: number of steps between logs [100000]
Option -learn: number of backprop learning steps [6000000]
Loading network configuration file ../models/LeNet.ini
Layer: conv1 [Conv(Frame_CUDA)]
Unreadable value for parameter Solvers.Clamping: "-1.0:1.0"
Notice: Unused section softmax.Target in INI file
Time elapsed: 0.806647 s
Error: Trying to read a negative number into an unsigned variable

And the same for LeNet_Spike:
../build/bin/n2d2 "../models/LeNet_Spike.ini" -test
Option -test: perform testing
Loading network configuration file ../models/LeNet_Spike.ini
Layer: conv1 [Conv(Transcode_CUDA)]
Unreadable value for parameter Solvers.Clamping: "-1.0:1.0"
Notice: Unused section fc2.Target in INI file
Notice: Unused section fc2.config in INI file
Time elapsed: 0.806895 s
Error: Trying to read a negative number into an unsigned variable

**Comment by olivierbichler-cea** [comment]
Closing the issue, please feel free to reopen it if necessary.

**Comment by dipsankarb** [comment]
./../build/bin/n2d2 "mnist24_16c4s2_24c5s2_150_10_Spike.ini" -test
Option -test: perform testing
Loading network configuration file mnist24_16c4s2_24c5s2_150_10_Spike.ini
Layer: conv1 [Conv(Transcode)]
Notice: Could not open configuration file: conv1.cfg
  # Shared synapses: 256
  # Virtual synapses: 30976
  # Inputs dims: 24 24 1 
  # Outputs dims: 11 11 16 
Layer: conv2 [Conv(Transcode)]
Notice: Could not open configuration file: conv2.cfg
  # Shared synapses: 2250
  # Virtual synapses: 36000
  # Inputs dims: 11 11 16 
  # Outputs dims: 4 4 24 
Layer: fc1 [Fc(Transcode)]
Notice: Could not open configuration file: fc1.cfg
  # Synapses: 57600
  # Inputs dims: 4 4 24 
  # Outputs dims: 1 1 150 
Layer: fc2 [Fc(Transcode)]
Notice: Could not open configuration file: fc2.cfg
  # Synapses: 1500
  # Inputs dims: 1 1 150 
  # Outputs dims: 1 1 10 
Target: fc2 (target value: 1 / default value: 0 / top-n value: 1)
Remove extra Reshape...
Total number of neurons: 2480
Total number of nodes: 2480
Total number of synapses: 61606
Total number of virtual synapses: 126076
Total number of connections: 126076
Notice: Unused section fc2.Target in INI file
GD Warning: PNG image support has been disabled
GD Warning: PNG image support has been disabled
Learning database size: 48000 images
Validation database size: 12000 images
Testing database size: 10000 images
